gui/resources.xml

```diff
@@ -1289,7 +1319,7 @@ Vocabulary
           <property name="icon-name">mypaint-tool-hsvsquare</property>
           <property name="label" translatable="yes" context="Menu→Color (labels), Accel Editor (labels)">HSV Square</property>
           <property name="tooltip" translatable="yes" context="Accel Editor (descriptions)">Dockable color selector: HSV square which can be rotated to show different hues.</property>
-          <!-- FIXME: this description needs to be improved -->
+          <!-- FIXME: this description needs to be improved Note-ALbert: Looks good to me.-->
```

Comment:
What about "A dockable HSV color selector which focuses the square's value and saturation range based on the position of the ring around the square."? Even though it is a mouthful, that dose explain it. 

We could probably go with @0ion9 suggestion, but we probably want to word the other HSV selectors with the same structure whichever direction we go.